src/components/Editor/index.js

```diff
@@ -141,6 +160,9 @@ class Editor extends PureComponent<Props, State> {
     const { codeMirror } = editor;
     const codeMirrorWrapper = codeMirror.getWrapperElement();
 
+    this.initPageVisibility.bind(this);
+    this.initPageVisibility();
```

Comment:
do we need the bind if we're using the assignment syntax `initPageVisibility = () => {}`

Reply:
method.bind(this) returns a new method but doesn't modify the existing one, so I think the first statement is simply not needed here?

Reply:
Thanks for the help from both of you :) Yes, that line seems to be unneeded after testing without it.